apps/web/src/app/(private)/projects/[projectId]/versions/[commitUuid]/documents/[documentUuid]/evaluations/[evaluationId]/_components/EvaluationResults/EvaluationResultInfo/index.tsx

```diff
@@ -83,23 +84,35 @@ export function EvaluationResultInfo({
   evaluationResult: EvaluationResultWithMetadata
   providerLog?: ProviderLogDto
 }) {
+  const ref = useRef<HTMLDivElement>(null)
   const [selectedTab, setSelectedTab] = useState<string>('metadata')
   const [selected, setSelected] = useState<MaybeDocumentLog>(null)
   const onClickOpen = useCallback(async () => {
     setSelected(evaluationResult.documentLogId)
   }, [evaluationResult.documentLogId])
+  const height = useDynamicHeight({ ref, paddingBottom: 16 })
   return (
     <>
-      <div className='w-80 flex-shrink-0 flex flex-col border border-border rounded-lg px-4 pt-6 items-center'>
-        <TabSelector
-          options={[
-            { label: 'Metadata', value: 'metadata' },
-            { label: 'Messages', value: 'messages' },
-          ]}
-          selected={selectedTab}
-          onSelect={setSelectedTab}
-        />
-        <div className='flex flex-col relative w-full h-full max-h-full max-w-full overflow-auto'>
+      <div
```

Comment:
All this wrapper classes are supper similar between

1. Evaluation log info side panel
2. Evaluation original log modal panel
3. Log sidebar info panel
<img width="405" alt="image" src="https://github.com/user-attachments/assets/a798bce3-b8d5-47ad-86ad-6d6415078f6f">

Probably is there an abstraction. For now I'll leave un-DRY

Reply:
now that you are at it maybe we can implement the correct design for the original log design 
<img width="360" alt="image" src="https://github.com/user-attachments/assets/b17aba65-4fbf-4fa2-9329-be50f14b30de">